app/assets/stylesheets/group.scss

```diff
@@ -0,0 +1,49 @@
+$white: #ffffff;
+
+.title {
+  color: $white;
+  font-size: 60px;
+  font-size: 5vw;
+  font-weight: bold;
+}
+
+.groups {
+  text-align: center;
+}
+
+.group {
+  background-color: rgba(244, 162, 23, 0.66);
+  -moz-border-radius: 15px;
+  -webkit-border-radius: 15px;
+  border-radius: 15px;
+  margin: 20px;
+  max-width: 1000px;
+  padding: 30px;
+}
+
+.group-title {
+  background-color: rgba(#144bd9, 0.6);
```

Comment:
Color literals like `#144bd9` should only be used in variable declarations; they should be referred to via variable everywhere else.